mkdocs/docs/how-to-release.md

```diff
@@ -84,14 +84,14 @@ Update the version in `pyproject.toml` and `pyiceberg/__init__.py` to match the
 * Focuses on critical bug fixes or security patches that maintain backward compatibility.
 * Examples: `0.8.1`, `0.8.2`.
 
-To create a patch branch from the latest release tag:
+To create a patch from the latest release tag:
 
 ```bash
 # Check out the base branch for the patch version
 git checkout pyiceberg-0.8.x
```

Comment:
@Fokko did you create this manually? if so I can edit the instructions to create the branch

Reply:
Good one! I've created this from the tag right away:

```sh
git checkout -b pyiceberg-0.8.x pyiceberg-0.8.0
```